scripts/fast-reboot

```diff
@@ -295,38 +259,26 @@ if [[ "$REBOOT_TYPE" = "fast-reboot" ]]; then
 fi
 
 # Kill swss dockers
-docker kill swss
-
-
-# Warm reboot: dump state to host disk
-if [[ "$REBOOT_TYPE" = "fastfast-reboot" ]]; then
-    mkdir -p $WARM_DIR
-
-    # Dump route table form APPL DB.
-    # This route table will be used by fpmsyncd
-    # reconcialtion logic
-    cleanup_except_table 0 'ROUTE_TABLE'
-    cleanup_except_table 4 'WARM_RESTART_TABLE'
-    cleanup_except_table 6 'WARM_RESTART_TABLE'
-
-    redis-cli -n 1 FLUSHDB
-    redis-cli -n 2 FLUSHDB
-    redis-cli -n 5 FLUSHDB
-
-    redis-cli save
-    docker cp database:/var/lib/redis/$REDIS_FILE $WARM_DIR
-    docker exec -i database rm /var/lib/redis/$REDIS_FILE
-fi
+docker kill swss > /dev/null
 
 # Pre-shutdown syncd
-if [[ "$REBOOT_TYPE" = "warm-reboot" ]]; then
+if [[ "$REBOOT_TYPE" = "warm-reboot" || "$REBOOT_TYPE" = "fastfast-reboot" ]]; then
     initialize_pre_shutdown
 
     request_pre_shutdown
 
     wait_for_pre_shutdown_complete_or_fail
 
-    backup_datebase
+    # Warm reboot: dump state to host disk
+    if [[ "$REBOOT_TYPE" = "fastfast-reboot" ]]; then
+        redis-cli -n 1 FLUSHDB &> /dev/null
```

Comment:
>FLUSHDB [](start = 23, length = 7)

Similar flush code is in files files/scripts/{swss,syncd}.sh. Could you add your branch there?

Reply:
I did not put flush there because of two reasons
1. if system warm restart flag is enabled and platform mellanox swss.sh flushes counters db so it will affect docker level swss restart
2. I don't think there is need to backup something that will be flushed later